### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by jasmingacic**
@edevenport I've added "FOG_MOCK" environment variable so running unit tests with `rake test` can be done both *real* and *mock*.

### CODE REVIEW

**Comment by jasmingacic** [comment]
I've added memberships to this PR

File: lib/packet/models/memberships.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require_relative "notification"
```
Comment (edevenport) [inline lib/packet/models/memberships.rb]: Wrong require; should be membership.

File: tests/packet/models/test_memberships.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+require_relative "../../test_helper.rb"
+
+# TestInvitations
+class TestInvitations < Minitest::Test
```
Comment (edevenport) [inline tests/packet/models/test_memberships.rb]: Wrong class name; should be memberships.

File: tests/packet/models/test_notifications.rb [hunk 3]
```diff
@@ -1,17 +1,11 @@
...
 class TestProjects < Minitest::Test
-  def self.test_order
-    :alpha
-  end
-
-  def setup
-    # Establish Connection
-    @compute = Fog::Compute::Packet.new(:packet_token => ENV["PACKET_TOKEN"])
-  end
+  # def setup
```
Comment (edevenport) [inline tests/packet/models/test_notifications.rb]: Can we remove the commented code?

File: tests/packet/requests/test_memberships.rb [hunk 4]
```diff
@@ -0,0 +1,43 @@
+require_relative "../../test_helper.rb"
+
+# TestInvitations
+class TestInvitations < Minitest::Test
```
Comment (edevenport) [inline tests/packet/requests/test_memberships.rb]: Wrong class name again.

File: tests/packet/requests/test_memberships.rb [hunk 5]
```diff
@@ -0,0 +1,43 @@
+require_relative "../../test_helper.rb"
+
+# TestInvitations
```
Comment (edevenport) [inline tests/packet/requests/test_memberships.rb]: Wrong comment name.

**Pull request merged**